triage/render.js

```diff
@@ -128,66 +172,75 @@ function renderCluster(top, key, keyId, text, tests) {
     return count == 1 ? count + ' ' + word : count + ' ' + word + suffix;
   }
 
+  var counts = clustered.makeCounts(keyId);
+
   var clusterSum = clustersSum(tests);
   var recentCount = clustered.getHitsInLastDayById(keyId);
   var failureNode = addElement(top, 'div', {id: keyId}, [
-    createElement('h2',
-      {innerHTML: `${plural(clusterSum, 'FAILURE', 'S')} (${recentCount} RECENT) MATCHING <a href="#${keyId}" class="key">${keyId}</a>`}),
+    createElement('h2', null, [
+      `${plural(clusterSum, 'FAILURE', 'S')} (${recentCount} RECENT) MATCHING `,
```

Comment:
I do not understand what recent means. Can we change this to today? Sounds more clear.